runtime/ast/statement.go

```diff
@@ -597,7 +597,9 @@ func (s *SwitchStatement) Accept(visitor Visitor) Repr {
 func (s *SwitchStatement) Walk(walkChild func(Element)) {
 	walkChild(s.Expression)
 	for _, switchCase := range s.Cases {
-		walkChild(switchCase.Expression)
+		if switchCase.Expression != nil {
```

Comment:
Maybe add a comment here
```suggestion
		// The default case has no expression
		if switchCase.Expression != nil {
```